sequencing/vcf_sitesonly/Readme.md

```diff
@@ -1,21 +1,18 @@
 <!-- dx-header -->
-# VCF QC (DNAnexus Platform App)
+# Get Sites-only VCF 
 
-Generates a sites-only file from full VCF files.
+## What does this app do?
 
-This is the source code for an app that runs on the DNAnexus Platform.
-For more information about how to run or modify it, see
-https://wiki.dnanexus.com/.
-<!-- /dx-header -->
+This app generates a 'sites-only' file from VCF file(s) (`*.vcf.gz`).
 
+## What data are required for this app to run?
 
+This app requires VCF file(s) (`*.vcf.gz`) as an input. 
 
-<!--
-TODO: This app directory was automatically generated by dx-app-wizard;
-please edit this Readme.md file to include essential documentation about
-your app that would be helpful to users. (Also see the
-Readme.developer.md.) Once you're done, you can remove these TODO
-comments.
+## What does this app output?
 
-For more info, see https://wiki.dnanexus.com/Developer-Portal.
--->
+This app outputs a site only VCF file(s) (`*.vcf.gz`) along with this index file. This output VCF file (`*.vcf.gz`) would contain the first eight columns of the original VCF file.
```

Comment:
This app outputs a site-only VCF file(s) (`*.vcf.gz`) along with its index file. The output VCF file (`*.vcf.gz`) will contain the first eight columns of the original VCF file.